common/religion/religions/doombringer.txt

```diff
@@ -24,6 +24,20 @@
 	doctrine = doctrine_deviancy_accepted
 	doctrine = doctrine_witchcraft_accepted
 
+	#Magic
+	doctrine = doctrine_light_magic_crime
+	doctrine = doctrine_shadow_magic_crime
```

Comment:
Actually, the Legion uses Shadow magic so I would make it accepted.
https://wow.gamepedia.com/Void#Demons

Reply:
Done